include/aws/common/json.h

```diff
@@ -6,10 +6,10 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-#include <aws/common/byte_buf.h>
 #include <aws/common/common.h>
 
 struct aws_json_value;
+struct aws_byte_buf;
 
```

Comment:
can't remove `#include <aws/common/byte_buf.h>`

we pass `struct aws_byte_cursor` by value in this file, so we need to know its actual size